sift/client.py

```diff
@@ -77,35 +90,42 @@ def track(
                 event name such as "$transaction" or "$create_order" or a custom event
                 name (that does not start with a $).
 
-            properties: A dict of additional event-specific attributes to track
+            properties: A dict of additional event-specific attributes to track .
 
             return_score: Whether the API response should include a score for this
-                 user (the score will be calculated using this event).  This feature must be
-                 enabled for your account in order to use it.  Please contact
-                 support@siftscience.com if you are interested in using this feature.
+                 user (the score will be calculated using this event).
 
             return_action: Whether the API response should include actions in the response. For
                  more information on how this works, please visit the tutorial at:
-                 https://siftscience.com/resources/tutorials/formulas
+                 https://siftscience.com/resources/tutorials/formulas .
```

Comment:
Yes -- this is to help ensure that browsers/editors/etc. that turn URLs into clickable links will point to [https://siftscience.com/resources/tutorials/formulas] and not [https://siftscience.com/resources/tutorials/formulas.].